Algorithms/arrays/leftShift/README.md

```diff
@@ -0,0 +1,53 @@
+# Algorithms
+
+## METHOD 1 (NAIVE) <br>
```

Comment:
```suggestion
## Method 1 (Naive)
```

- For all headings and sub-headings, make sure to follow `title case`, and not upper case. (**Change it for `Method 2` and `Method 3` sub-headings as well**)
- The `<br>` is not required in markdown, as it automatically renders with a blank space, for headings. (**Change it for `Method 2` and `Method 3` sub-headings as well**)